llvm/include/llvm/IR/IntrinsicsAArch64.td

```diff
@@ -3095,23 +3095,23 @@ let TargetPrefix = "aarch64" in {
                             [llvm_anyvector_ty, LLVMMatchType<0>, LLVMMatchType<0>, LLVMMatchType<0>],
                             [IntrNoMem]>;
 
-  class SME2_CVT_FtoI_VG2_Intrinsic
+  class SME2_CVT_ItoF_VG2_Intrinsic
```

Comment:
nit: while we're changing this, for this one and the others:
```suggestion
  class SME2_CVT_ItoF_X2_Intrinsic
```

Reply:
Done